src/query.c

```diff
@@ -571,10 +581,11 @@ static IndexIterator *Query_EvalPrefixNode(QueryEvalCtx *q, QueryNode *qn) {
     return NULL;
   }
 
-  rune *str = NULL;
   size_t nstr;
-  if (qn->pfx.tok.str) {
-    str = strToFoldedRunes(qn->pfx.tok.str, &nstr);
+  rune *str = strToFoldedRunes(qn->pfx.tok.str, &nstr);
+  if (!str) {
+    QueryError_SetErrorFmt(q->status, QUERY_ELIMIT, "%s query string is too long", PrefixNode_GetTypeString(&qn->pfx));
```

Comment:
Can we also include the limit in the message?  (value of `MAX_RUNESTR_LEN`)
(although we would also document it)
(also relevant for the other message below)